### PR TITLE
Updated Community Rules to discourage TOS breaking of other platforms.

### PR DESCRIPTION
**Pull request by JPVenson**
Added TOS section in Rules

### CODE REVIEW

**Comment by felix920506** [comment]
You should add a changelog section at the bottom of the page

**Comment by felix920506** [comment]
Also, isn't the ToS of other platforms part already covered in another section of the rules?

**Comment by JPVenson** [comment]
> Also, isn't the ToS of other platforms part already covered in another section of the rules?

I dont think so to be frank, thats why i added it^^

File: docs/general/community-standards/index.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ At all times when interacting with the Jellyfin community via any method (Matrix
 
 - **Do not engage in, encourage, facilitate or discuss piracy in any Jellyfin communities.** Jellyfin is a media server system for your own media collection; how you obtain media is not our concern and is not to be discussed in our communities in any form. **Do not ask about where or how to obtain media**, **do not ask about, sell, trade, or otherwise facilitate access to other users' servers** or **discuss any piracy related topics. This includes discussing technologies commonly used for piracy and personal philosophies about it**, in our communities. Please also refrain from discussing any topic that is closely related to piracy or accompanying topics and technology.
 
+- **Do not engage in, encourage, facilitate or discuss anything that will break the Terms Of Service of other Platforms.** Jellyfin lives in an Ecosystem where it is important to keep a balance of user features and the impact on other systems. It is our goal to **not** leave an impact or obtaining a reputation for allowing to use Jellyfin in conjunction with other systems, that were not designed or allowed to be used with jellyfin to keep other Projects alive and reserved for their intended scopes.
```
Comment (Shadowghost) [inline docs/general/community-standards/index.md:28]:
```suggestion
- **Do not engage in, encourage, facilitate or discuss anything that will break the Terms of Service of other platforms.** Jellyfin lives in an ecosystem where it is important to keep a balance of user features and the impact on other systems. It is our goal to **not** leave an impact or obtaining a reputation for allowing to use Jellyfin in conjunction with other systems, that were not designed or allowed to be used with Jellyfin to keep other Projects alive and reserved for their intended scopes.
```

File: docs/general/community-standards/index.md [hunk 2]
```diff
@@ -25,6 +25,8 @@ At all times when interacting with the Jellyfin community via any method (Matrix
 
 - **Do not engage in, encourage, facilitate or discuss piracy in any Jellyfin communities.** Jellyfin is a media server system for your own media collection; how you obtain media is not our concern and is not to be discussed in our communities in any form. **Do not ask about where or how to obtain media**, **do not ask about, sell, trade, or otherwise facilitate access to other users' servers** or **discuss any piracy related topics. This includes discussing technologies commonly used for piracy and personal philosophies about it**, in our communities. Please also refrain from discussing any topic that is closely related to piracy or accompanying topics and technology.
 
+- **Do not engage in, encourage, facilitate or discuss anything that will break the Terms of Service of other platforms.** Jellyfin lives in an ecosystem where it is important to keep a balance of user features and the impact on other systems. It is our goal to **not** leave an impact or obtaining a reputation for allowing to use Jellyfin in conjunction with other systems, that were not designed or allowed to be used with Jellyfin to keep other Projects alive and reserved for their intended scopes.
```
Comment (joshuaboniface) [inline docs/general/community-standards/index.md:28]:
```suggestion
- **Do not engage in, encourage, facilitate or discuss Terms of Service (TOS) violations or suggest configurations that may violate the TOS of any other platforms.** Jellyfin can interface with multiple external services, both internally (e.g. Metadata provides) and by user configuration, each of which has its own requirements and TOS to protect itself. We do not want Jellyfin or its community to develop a reputation for facilitating circumvention or breaking of other services' TOSes or be seen as a problematic member of the wider community. The exact nature of these violations is subject to interpretation on a case-by-case basis based on the service in question and the discussion, and this rule is designed to bring attention to the issue. As a concrete example, the Cloudflare Terms Of Service forbid video streaming behind a normal Cloudflare tunnel; thus, suggesting a user run their instance in this way is a violation of this rule.
```

**Review by joshuaboniface — CHANGES_REQUESTED** [review]

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**